ethereum/permissioning/src/main/java/tech/pegasys/pantheon/ethereum/permissioning/node/NodePermissioningController.java

```diff
@@ -100,4 +101,11 @@ public long subscribeToUpdates(final Runnable callback) {
   public boolean unsubscribeFromUpdates(final long id) {
     return permissioningUpdateSubscribers.unsubscribe(id);
   }
+
+  public Optional<NodeLocalConfigPermissioningController> localConfigController() {
```

Comment:
We expose the NodeLocalConfigPermissioningController to supply it to the node whitelist JSON-RPC methods.